app/views/users/show.html.erb

```diff
@@ -1,8 +1,90 @@
-<h3><%= @user.name %>'s Dashboard</h3>
-
-<%= button_to "Discover Movies", user_discover_path(@user), method: :get %>
-
+<div style="text-align: center;">
+  <h2><%= @user.name %>'s Dashboard</h2>
+  <%= button_to "Discover Movies", user_discover_path(@user), method: :get %></div>
 <div id="my_parties">
-<h5>My Parties</h5>
-
-</div>
\ No newline at end of file
+  <h3>My Parties</h3>
+</div>
+<div id='invited-parties'>
+  <h3>Parties I'm Hosting</h3>
+  <% @hosted_parties.each do |party| %>
+    <h3>Viewing Party ID: <%= party.id %></h3>
+    <table border="1">
+      <thead>
+        <tr>
+          <th>Movie Image</th>
+          <th>Movie Title</th>
+          <th>Date</th>
+          <th>Time</th>
+          <th>Host</th>
+          <th>Guests</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <% if @movie_images[party.movie_id].present? %>
+              <img src="https://image.tmdb.org/t/p/w500<%= @movie_images[party.movie_id] %>" alt="<%= @movies[party.movie_id] %>" width="100">
```

Comment:
Need method `movie_image` in `ViewingParty` model to avoid making this instance variable

Reply:
Done